src/models/user.js

```diff
@@ -135,6 +131,7 @@ const userSchema = new mongoose.Schema(
     },
     avaregeRating: {
       type: Number,
+      required: true,
```

Comment:
I think this shouldn't be required

Reply:
All the ones that have a `default` will not be a problem when using 3rd party auth. No need to specify them; they'll be added automatically, by default. I wanted to make them look unified. I still can change them if they're causing any issues.

Reply:
I vote for removing the required since they aren't used in this case.

Reply:
All the `required: true` fields that also have a `default` are only to make the model look unified; they can be removed. On the other hand, they do not really affect how the database functions.

I thought of the profile photo causing a problem when is removed, but there _will_ be a default, placeholder, image for the profile photo. I still can remove the `required: true` fields if you think they'll cause any issues.

Reply:
Removed them 👍🏽